eng/common/pipelines/templates/steps/validate-all-packages.yml

```diff
@@ -10,20 +10,21 @@ steps:
       displayName: "Set as release build"
       condition: and(succeeded(), eq(variables['SetAsReleaseBuild'], ''))
 
-    - task: Powershell@2
+    - task: AzureCLI@2
```

Comment:
An FYI only and not blocking: `AzureCLI@2` seems to better respect the last exit code over `AzurePowerShell@5`. 

There is an issue with `AzurePowerShell@5` where one has to explicitly exit or a failing script will be seen by DevOps as succeeding. 
 
No changes are needed here. Example pipeline runs showing `AzureCLI@2` is checking `$LASTEXITCODE` in powershell by default: https://dev.azure.com/azure-sdk/playground/_build/results?buildId=3856191&view=logs&j=5c65763d-3a87-52b5-f04f-2257d72cd1d6&t=6d42ae89-b318-56f3-6292-f16fc698725a